templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>
+      User: {{ user.username }}
+    </li>
+    <li>
+      <a href="{% url "logout" %}">Log out</a>
```

Comment:
The `logout` URL is correctly referenced. Ensure that the URL name `logout` is correctly defined in your authentication URLs.